amphora-image-builder/README

```diff
@@ -15,5 +15,10 @@ Usage
 -----
 
 ```
-$ ./build.sh
+./build.sh
+
+# Set as appropriate to your environment
+export TARGET_PROJECT=service
+
+openstack image create --property hw_disk_bus=virtio --property=virtio-scsi --property=hw_vif_multiqueue_enabled=true --private --project $TARGET_PROJECT --disk-format raw --container-format bare --file amphora-x64-haproxy.raw --progress amphora-x64-$(date +%Y-%m-%d)-haproxy
```

Comment:
is this line correct? the --property flags look like they're formatted differently